pkg/storaged/stratis-panel.jsx

```diff
@@ -52,7 +55,15 @@ function stratis_pool_row(client, path) {
 
     return {
         client,
-        name: pool.Name,
+        name: (
+            pool.AvailableActions === "fully_operational"
+                ? pool.Name
+                : (
+                    <Flex spaceItems={{ default: 'spaceItemsXs' }} alignItems={{ default: 'alignItemsCenter' }}>
+                        <Icon status="status"><ExclamationCircleIcon /></Icon>{pool.Name}
```

Comment:
There is an existing mechanism for getting the warning triangles in all kinds of places, see the `job_path`  parameter of `SidePanelRow`.

This mechanism is old and doesn't feel natural for non-UDisks things... but let's maybe add a `has_warning` parameter to `SidePanelRow`?